assets/src/edit-story/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -23,20 +23,22 @@ import {
   trackEvent,
   getTimeTracker,
 } from '@web-stories-wp/tracking';
+import {
+  createBlob,
+  getImageDimensions,
+  getFileName,
+} from '@web-stories-wp/media';
 
 /**
  * Internal dependencies
  */
 import { useUploader } from '../../../uploader';
 import useReduction from '../../../../utils/useReduction';
-import { createBlob } from '../../../../utils/blobs';
 import { noop } from '../../../../utils/noop';
-import getFileName from '../getFileName';
 import useUploadVideoFrame from '../useUploadVideoFrame';
 import useFFmpeg from '../useFFmpeg';
 import getResourceFromAttachment from '../getResourceFromAttachment';
 import getResourceFromLocalFile from '../getResourceFromLocalFile';
```

Comment:
^ Should these also be moved?

Reply:
See https://github.com/google/web-stories-wp/pull/7954#discussion_r651631899